### ISSUE TITLE
Clarification on some parameters

### ISSUE THREAD

**Issue by girum89** [Sprinter1999/FedLSR#1]
Hello and thank your for your research and code.
I have some question on the notebook provided here.

1.  Under the 'Client' cell, inside  model training to get ` logits1` and `logits2`  outputs from the model are multiplied by 3, can you comment on that?
2. The self entropy regularization  `L_e`  is it mentioned on the paper? and maybe a typo  logits1 is used in both sum operations.

Thanks :)

**Comment by Sprinter1999** [comment]
**Thank you Girum**! It's a huge pleasure for me to clarify your quesions on the DEMO version of this work!
1. The logits o_1 and o_2 are multiplied by 3, which indicates the self-distillation temperature is 1/3. I want to add an ablation experiment in the experiment section, but I find there's no enough space to place this ablation study (this year the full paper length is restrained to 9 pages), so this temperature is somewhat too straightforward and strange to be directly given for readers. In fact it's an **empirical hyper-parameter** T_d in our method and I just follow some related papers from NLP and other areas. Thank you for commenting on this which also gives me a chance to clarify in the Github Issue frontend.
2. Yes, you are right. **So I add this in the discussion section.** In fact, the reviewers ask me a question:"Does these two regularization enough to be robust?". I missed this issue when I rushed to complete this paper (I was very busy and depressed in April&May this year due to school works and other personal reasons). The entropy loss is given in the first sub-section of Method and Discussion sections, which is a common extra optimization term in many semi-supervised learning works which further forces the model to give more sharpened prediction on the basis of LSR. Both logits 1 and logits 2 are simultaneously used and we just take the average (In fact only o_1 or o_2 is also okay!). BTW, I sincerely thank reviewers for their comments.
3. I'll devote more effort to the extended version of this work. It will be a great honor for me if you have some interest in this work, maybe we can work together to do some intersting things together : ) 
**Thank you again** for your time and care for my first work, it's far away from perfect, but you encourage me a lot.

**Comment by girum89** [comment]
Thank you @Sprinter1999 for the quick response and clarification !! 
I am also starting my research in Federated Learning and hope to collaborate :)
